app/controllers/admin/data_update_scripts_controller.rb

```diff
@@ -9,6 +9,8 @@ def index
     def show
       response = DataUpdateScript.find(params[:id])
       render json: { response: response }
+    rescue StandardError => e
```

Comment:
**nitpick (non-blocking):** We catch `StandardError`, but return a status of `not_found`. How about we rescue the narrowest exception possible?

```suggestion
    rescue ActiveRecord::RecordNotFound=> e
```